frontend/app/routes/$lang+/_protected+/personal-information+/preferred-language+/confirm.tsx

```diff
@@ -1,125 +0,0 @@
-import type { ActionFunctionArgs, LoaderFunctionArgs, MetaFunction } from '@remix-run/node';
-import { json, redirect } from '@remix-run/node';
-import { Form, useLoaderData, useParams } from '@remix-run/react';
-
-import { useTranslation } from 'react-i18next';
-import { redirectWithSuccess } from 'remix-toast';
-
-import pageIds from '../../../page-ids.json';
-import { Button, ButtonLink } from '~/components/buttons';
-import { getInstrumentationService } from '~/services/instrumentation-service.server';
-import { getLookupService } from '~/services/lookup-service.server';
-import { getRaoidcService } from '~/services/raoidc-service.server';
-import { getUserService } from '~/services/user-service.server';
-import { getNameByLanguage, getTypedI18nNamespaces } from '~/utils/locale-utils';
-import { getFixedT } from '~/utils/locale-utils.server';
-import { mergeMeta } from '~/utils/meta-utils';
-import { getPathById } from '~/utils/route-utils';
-import type { RouteHandleData } from '~/utils/route-utils';
-import { getTitleMetaTags } from '~/utils/seo-utils';
-
-export const handle = {
-  breadcrumbs: [
-    { labelI18nKey: 'personal-information:preferred-language.confirm.breadcrumbs.personal-information', routeId: '$lang+/_protected+/personal-information+/index' },
```

Comment:
All the i18n keys that are used by this page can also be removed.

Reply:
fixed: 169392aaee2ead571759bb714d528b1001f60ace